app/src/main/java/org/mozilla/focus/settings/privacy/PrivacySecuritySettingsToolTip.kt

```diff
@@ -0,0 +1,114 @@
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mozilla.focus.R
+import org.mozilla.focus.ui.theme.FocusTheme
+import org.mozilla.focus.ui.theme.focusColors
+
+@Composable
+@Preview
+private fun ToolTipContentPreview() {
+    ToolTipContent {
+    }
+}
+
+/**
+ * Displays a tool tip for Privacy Security Settings screen.
+ *
+ * @param onDismissToolTip Callback when the "x" Button from tool tip is pressed.
+ */
+@Composable
+fun ToolTipContent(onDismissToolTip: () -> Unit) {
+    FocusTheme {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentSize(Alignment.Center)
+        ) {
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .wrapContentHeight()
+                    .padding(16.dp)
+                    .clip(RoundedCornerShape(10.dp))
+                    .background(
+                        shape = RoundedCornerShape(10.dp),
+                        brush = Brush.verticalGradient(
+                            colors = listOf(
+                                colorResource(R.color.cfr_pop_up_shape_end_color),
+                                colorResource(R.color.cfr_pop_up_shape_start_color)
+                            )
+                        )
+                    ),
+                contentAlignment = Alignment.CenterStart
+            ) {
+                Column(
+                    modifier = Modifier
+                        .wrapContentHeight()
+                        .padding(16.dp),
+                    verticalArrangement = Arrangement.spacedBy(
+                        10.dp
+                    )
+                ) {
+                    Text(
+                        text = stringResource(R.string.tool_tip_title),
+                        fontSize = 16.sp,
+                        letterSpacing = 0.5.sp,
+                        color = focusColors.privacySecuritySettingsToolTip,
+                        fontFamily = FontFamily(Font(R.font.metropolis_bold)),
+                        lineHeight = 20.sp
+                    )
+                    Text(
+                        text = stringResource(R.string.tool_tip_message),
+                        fontSize = 14.sp,
+                        letterSpacing = 0.5.sp,
```

Comment:
I think the letter spacing is only for the title.
@jeffreygee for confirmation.
Also @jeffreygee , can you confirm the message body text style? It seems it is not the same for all CFR messages. Thank you!

Reply:
Also, looking at the designs, the gradient has an angle, it's not horizontal or vertical. It might be more difficult obtaining that in Compose (would need a custom element) or you can try having it in an XML and setting it as painter resource.

In either case, @jeffreygee should confirm the result.

Reply:
@mcarare -  since this was a reused component that exists in Fenix right now - is there a way to crosscheck our CFR with theirs?

I was also looking at screenshot - the header looks correct (until we go get rid of Metropolis and completely to native fonts), but the message body should be the the Android native font (Roboto)